### PR TITLE
S3T13 Clean Up Event Page JS

### PR DESCRIPTION
**Pull request by mallowmew**

### CODE REVIEW

**Comment by mallowmew** [comment]
There isn't a line separator when each Event is shown in the list - @RachmannJoubert @Tibereon

File: public/js/EventsPage.js [hunk 1]
```diff
@@ -1,15 +1,111 @@
-//Adds event listener to submit button that gets data from form and validates it.
-document.getElementById('submit-event').addEventListener('click', e => {
+const eventList = document.querySelector('#events')
+const eventForm = document.querySelector('form')
+const message = document.querySelector('#messages')
+
+/**
+ * Gets event information from the API and passes into the displayHandler function
```
Comment (lizsharp) [inline public/js/EventsPage.js]: docblock - should be displayEventsHandler

File: public/js/EventsPage.js [hunk 2]
```diff
@@ -1,15 +1,111 @@
...
+        if (event.notes !== null) {
+            eventInformation += `<p>Notes: ${event.notes}</p>`
+        }
+        eventInformation += `</div></div>`
+    })
+    eventList.innerHTML = eventInformation
+
+    let showInfoButtons = document.querySelectorAll('.show-event-info')
+    showInfoButtons.forEach(function (button) {
+        button.addEventListener('click', (e) => {
```
Comment (lizsharp) [inline public/js/EventsPage.js]: No parenthesis required around (e) if not async.

File: public/js/EventsPage.js [hunk 3]
```diff
@@ -1,15 +1,111 @@
...
+}
+
+/**
+ * Runs a foreach through each event object and outputs HTML elements with event details
+ *
+ * @param events is an array of objects which contains information about events
+ */
+function displayEventsHandler(events) {
+    let eventInformation = ''
+    events.forEach(event => {
```
Comment (FelixLetheren) [inline public/js/EventsPage.js]: nice chuncky boi.

File: templates/EventsPage.phtml [hunk 4]
```diff
@@ -6,9 +6,7 @@
     <title>Academy Events</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <link rel="stylesheet" type="text/css" href="./css/genericStyles.css">
-    <script defer src='./js/EventsPage.js'></script>
-    <script defer src='./js/getEvents.js'></script>
-    <script defer src="./js/addEvent.js"></script>
+    <script defer src='./js/eventsPage.js'></script>
```
[the remark this answers is not in the document]
Comment (mallowmew) [reply templates/EventsPage.phtml]: I did change this locally but it seems like git doesn't care about casing

**Comment by RachmannJoubert** [comment]
> There isn't a line separator when each Event is shown in the list - this is different from how things look on the hiring partner page
> @RachmannJoubert @Tibereon

All sorted

**Pull request merged**